src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/SupplierAssessmentController.kt

```diff
@@ -24,13 +26,15 @@ class SupplierAssessmentController(
   ): AppointmentDTO {
     val user = userMapper.fromToken(authentication)
     val supplierAssessment = supplierAssessmentService.getSupplierAssessmentById(id)
-
+    appointmentValidator.validateUpdateAppointment(updateAppointmentDTO)
     return AppointmentDTO.from(
       supplierAssessmentService.createOrUpdateSupplierAssessmentAppointment(
         supplierAssessment,
         updateAppointmentDTO.durationInMinutes,
         updateAppointmentDTO.appointmentTime,
-        user
+        user,
+        updateAppointmentDTO.appointmentDeliveryType,
+        updateAppointmentDTO.appointmentDeliveryAddress
```

Comment:
I couldn't find tests around passing this address on from the controller to SupplierAssessmentService to AppointmentService -- is this tested in an integration test?

Reply:
I've now added address into the SupplierAssessmentControllerTest